DemoServer/Controllers/Demos/StaticIndexes/StoreFieldsOnIndex/metadata.json

```diff
@@ -0,0 +1,248 @@
+                "",
+                "* These indexed terms can later be queried on to get their matching documents, the documents that contain them.",
+                "",
+                "* In this example, field `TotalItemsOrdered` is computed and will be stored on the index in the next step."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "Store field on Index",
+            "Slug": "step-5",
+            "DescriptionLines": [
+                "* Use `Store.Add()` to store a field on the index.",
+                "",
+                "* At query time, the value will be fetched directly from the index instead of from the document store.",
+                "",
+                "* Note: Storing fields on the index will increase the index size on disk."
+            ],
+            "Assets": [
+                {
+                    "Title": "Storing Data in the Index",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#storing-data-in-the-index"
+                },
+                {
+                    "Title": "Indexes: Storing Data",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/5.3/Csharp/indexes/storing-data-in-index"
+                },
+                {
+                    "Title": "Studio: Index Field Options",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/studio/database/indexes/create-map-index#index-field-options"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug" : "step-6",
+            "DescriptionLines": [
+                "* Query the 'Orders' collection using the defined index.",
+                "",
+                "* The orders' documents can be _filtered_ by the index-fields defined in `OrdersQuantity_ByCompany.IndexEntry`",
+                "",
+                "* The resulting documents will include only orders that where made by 'companyId'."
```

Comment:
It may be a good idea to mentioned where `companyId` is coming from (input at the top of the page)

Reply:
See https://issues.hibernatingrhinos.com/issue/HRINT-352#focus=Comments-67-201202.0-0